ipaserver/plugins/dogtag.py

```diff
@@ -256,7 +256,7 @@
 from ipapython import dogtag, ipautil, certdb
 from ipaserver.masters import find_providing_server
 
-if api.env.in_server:
+if api.isdone('finalize') and api.env.in_server:
```

Comment:
Is this import check still necessary? ``freeipa-server`` depends on ``python3-pki``, so the import should never fail.

Reply:
I suspect this is legacy code from when the plugins were included in the client. I'll remove it.